scenario/src/main/kotlin/com/cosmotech/scenario/service/ScenarioServiceImpl.kt

```diff
@@ -712,8 +720,16 @@ internal class ScenarioServiceImpl(
     }
 
     if (scenario.security != null && existingScenario.security == null) {
```

Comment:
remove and prevent modification of security. security modification should be done in endpoint security